extensions/positron-r/src/r-installation.ts

```diff
@@ -143,6 +149,12 @@ export class RInstallation {
 		this.current = current;
 		this.reasonDiscovered = reasonDiscovered;
 
+		// Check if the installation is the default R interpreter
```

Comment:
```suggestion
		// Check if the installation is the default R interpreter for Positron
```

I think this will be helpful to remind anyone who comes later that this does not necessarily have anything to do with the system's current or default version of R. In fact, it's almost the opposite? Because if it's OK to use the system's default version of R, you don't need this setting.